holoviews/core/spaces.py

```diff
@@ -410,9 +417,12 @@ class Callable(param.Parameterized):
     inputs = param.List(default=[], doc="""
          The list of inputs the callable function is wrapping.""")
 
-    def __init__(self, **params):
+    def __init__(self, callable_function=None, stream_mapping={}, **params):
+        if callable_function is not None:
+            params['callable_function'] = callable_function
```

Comment:
>Any reason stream_mapping can't just be a parameter, allowing us to go back to the def __init__(self, **params) constructor?

Agreed, not sure why I didn't make it one.

>In addition, how can callable_function be None? Isn't it required to define a valid Callable?

The parameter currently defines a no-op, which this leaves unchanged, but thinking about it, I don't think ``callable`` should just be None and become a required first argument.